python/modem-firmware-1.3+/nrf_cloud_device_mgmt.py

```diff
@@ -118,10 +120,10 @@ class nRFCloudDevice:
     sub_type = ''
 
     def __repr__(self):
-        dev_str = '{}, {}, App[{}, {}, {}], Modem[{}, {}], Boot[{}, {}], Tags{}'.format(
+        dev_str = '{}, {}, App[{}, {}, {}], Modem[{}, Delta:{} Full:{}], Boot[{}, {}], Tags{}'.format(
             self.id, self.name,
             self.app_name, self.app_ver, self.app_fota,
-            self.mfw_ver, self.mfw_fota,
+            self.mfw_ver, self.mfw_delta_fota, self.mfw_full_fota,
             self.boot_ver, self.boot_fota,
             self.tags)
         return dev_str
```

Comment:
I think this would be a good use-case for fstrings:

```suggestion
       return 	f'{self.id}, {self.name}, '
				f'App[{self.app_name}, {self.app_ver}, {self.app_fota}], '
				f'Modem[{self.mfw_ver}, Delta:{self.mfw_delta_fota} Full:{mfw_full_fota}], '
				f'Boot[{self.boot_ver}, {self.boot_fota}], '
				f'Tags{self.tags}'
```

Reply:
Correction, sorry, parentheses are needed on the return statement to deal with Python's strange indentation requirements:


```py
 return (
	f'{self.id}, {self.name}, '
	f'App[{self.app_name}, {self.app_ver}, {self.app_fota}], '
	f'Modem[{self.mfw_ver}, Delta:{self.mfw_delta_fota} Full:{mfw_full_fota}], '
	f'Boot[{self.boot_ver}, {self.boot_fota}], '
	f'Tags{self.tags}'
```)